plotly/dashboard_objs/dashboard_objs.py

```diff
@@ -440,7 +434,7 @@ def insert(self, box, side='above', box_id=None):
         my_dboard.insert(box_1, 'left', 1)
         my_dboard.insert(box_1, 'below', 2)
         my_dboard.insert(box_1, 'right', 3)
-        my_dboard.insert(box_1, 'above', 4)
+        my_dboard.insert(box_1, 'above', 4, fill_percent=30)
```

Comment:
<img width="466" alt="screen shot 2018-02-28 at 11 29 07 am" src="https://user-images.githubusercontent.com/2789078/36799373-0b56d2a4-1c7b-11e8-932f-366992196f2a.png">

does this output for the example look correct to you @Kully ? Not sure `right/left` or `above/below` is working

Reply:
Running the exact same code in my branch, I am getting this preview:
![screen shot 2018-02-28 at 3 22 48 pm](https://user-images.githubusercontent.com/10369095/36811317-542418f0-1c9b-11e8-9c94-6a83568b4588.png)

What could be different about our branches? (you are cloning from the latest commit I assume)

Reply:
@Kully running on your branch in python 2.7 I get the desired output but in python 3.6 I get: 
![image](https://user-images.githubusercontent.com/2789078/36813837-cf868cb4-1ca3-11e8-80ef-8fed760ec441.png)
We should make this consistent across python versions.

Reply:
wow, that's interesting -
I'll investigate that (and probably need to add better tests)